TKGame/TKGame.cs

```diff
@@ -152,11 +155,24 @@ protected override void Draw(GameTime gameTime)
             // Draw each wall to the screen
             foreach (Wall wall in walls)
             {
-                spriteBatch.Draw(wall.Texture, wall.Rect, Color.Beige);
+                spriteBatch.Draw(wall.Texture, wall.Rect, new Color(0x9a, 0x9b, 0x9c, 0xFF));
```

Comment:
Hard coded?

Reply:
Yeah, I didn't know what color we wanted the walls, and none of the default colors were very pog. I'll slap it into a variable.